test/alternator/test_scylla.py

```diff
@@ -31,15 +33,15 @@ def test_localnodes(scylla_only, dynamodb):
 
 # The "this_dc" fixture figures out the name of Scylla DC to which "dynamodb"
 # is connected. Any test using this fixture automatically becomes scylla_only.
-@pytest.fixture(scope="session")
+@pytest.fixture(scope=testpy_test_fixture_scope)
```

Comment:
Here too, I think the fixtures in this file should have "module" scope, not session.